docs/source/membership/membership.md

```diff
@@ -172,11 +172,11 @@ A local MSP folder contains the following sub-folders:
 
   **Note:** If you are using a [Hardware Security Module(HSM)](../hsm.html) for key management, this folder is empty because the private key is generated by and stored in the HSM.
 
-* **signcert:** For a peer or orderer node (or in a client's local MSP) this folder contains the node's **signing key**. This key matches cryptographically the node's identity included in **Node Identity** folder and is used to sign data --- for example to sign a transaction proposal response, as part of the endorsement phase.
+* **signcert:**  For a peer or orderer node (or in a client’s local MSP) this folder contains the node’s certificate issued by CA. This certificate contains the node's **public key**. While private key is used to digitally sign a transaction, public key is used to verify that the signature attached to the transaction is valid.
 
-  This folder is mandatory for local MSPs, and must contain exactly one public key. Obviously, access to this folder must be limited only to the identities of users who have  administrative responsibility on the peer.
+  This folder is mandatory for local MSPs, and must contain exactly one public key. Obviously, access to this folder must be with all the members of a channel.
```

Comment:
Why do you think access to this folder must be with all members of a channel and not limited to admin identities?

Reply:
Since every transaction within a block is validated at each peer during Phase 3 -Validation and Commit of Transaction Flow, each peer on the channel must have access to public key inside signcert folder to be able to verify the transactions.

Please correct me if my understanding is incorrect.

Reply:
This is incorrect.

Standard X.509 certificates operate based on a chain of trust.  Loosely speaking:

A node generates a public/private key pair, and creates a 'certificate signing request' (CSR) containing the public key and some other information.  It sends this request to the CA, who creates a certificate for the node, and signs it.  This certificate refer to the 'issuing CA', and includes the CA's signature.

When it comes time to validate a transaction (or any other action secured by a cryptographic signature), the verifier first extracts the certificate/identity from the message, then checks to see if the issuing CA is known to and trusted by the verifier.  If so, the verifier confirms that the signature from the issuing CA is valid.  At this point we know that the certificate itself is valid, then we can verify that the signature over the message is in fact from this certificate.

So, in general no, the signcert folder is not shared/known to the whole network.  Instead, the signcert certificate is generally embedded along with the signature, and then the chain of trust from the issuing CAs is used to validate the certificate's authenticity.

Reply:
Understood. Thanks for the explanation.